### PR TITLE
Raise exception when override option is invalid

### PR DESCRIPTION
**Pull request by Marlin-Na**

### CODE REVIEW

File: canine/localization/base.py [hunk 1]
```diff
@@ -464,10 +464,12 @@ def prepare_job_inputs(self, jobId: str, job_inputs: typing.Dict[str, str], comm
                                 'download',
                                 value
                             )
-                    elif mode is None:
+                    elif mode is None or mode == 'null':
```
Comment (julianhess) [inline canine/localization/base.py]: What is the reason for adding `mode == "null"` synonymous to `mode == None`?

File: canine/localization/base.py [hunk 2]
```diff
@@ -464,10 +464,12 @@ def prepare_job_inputs(self, jobId: str, job_inputs: typing.Dict[str, str], comm
...
                         # Do not reserve path here
                         # null override treats input as string
                         self.inputs[jobId][arg] = Localization(None, value)
+                    else:
+                        raise ValueError("Invalid override option [{}]".format(mode))
```
Comment (julianhess) [inline canine/localization/base.py]: Thanks for fixing this oversight!

**Comment by Marlin-Na** [comment]
yaml's null will be translated to python's None. However string 'null' in
yaml will not. Thought it would be better to give a string option as well.

julianhess <notifications@github.com> 于 2020年2月21日周五 下午4:14写道：

**Comment by julianhess** [comment]
Makes sense! I forgot about the use case of YAML-serialized workflows.

**Pull request merged**